schedule_lessons/dashboard/urls.py

```diff
@@ -2,6 +2,7 @@
 from django.urls import path
 from . import views
 
+
```

Comment:
nit: remove this newline

Reply:
PEP8 standards and linting asks to leave two lines after import statements